cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/ISO8601Date.java

```diff
@@ -0,0 +1,26 @@
+package com.cloudesire.platform.apiclient;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class ISO8601Date
+{
+    private final Date date;
+
+    public ISO8601Date( Date date )
+    {
+        this.date = date;
+    }
+
+    @Override
+    public String toString()
+    {
+        final SimpleDateFormat dateFormat = new SimpleDateFormat( "yyyy-MM-dd" );
+        return dateFormat.format( date );
```

Comment:
we can obtain the `DateFormat` from a method specific to every class in hierarchy and avoid the override of the `toString` in every child class